src/app/components/search/SearchKeyword.js

```diff
@@ -155,7 +155,7 @@ class SearchKeyword extends React.Component {
 
   /*
   check that the sort parameter is the default, if it is, set the parameter to "clear"
-  to identify that no other sort filter is applied and the search query should be reseted
```

Comment:
FYI: It felt more like a real PR if I fixed a typo to make it a two line PR instead of just one